From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: xiaoyinl <xiaoyinl@users.noreply.github.com>
Date: Tue, 30 Dec 2025 22:29:37 +0800
Subject: [PATCH] 8838: Add more Program Starting Locations

Add wmain, wWinMain, and DriverEntry functions as starting locations for Windows PE files.
---
 .../plugin/core/navigation/ProgramStartingLocationOptions.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/navigation/ProgramStartingLocationOptions.java b/Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/navigation/ProgramStartingLocationOptions.java
index 64182f2aaf..d505c76edd 100644
--- a/Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/navigation/ProgramStartingLocationOptions.java
+++ b/Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/navigation/ProgramStartingLocationOptions.java
@@ -60,7 +60,7 @@ public class ProgramStartingLocationOptions implements OptionsChangeListener {
 			"a newly discovered starting symbol, provided the user hasn't manually moved.";
 
 	private static final String DEFAULT_STARTING_SYMBOLS =
-		"main.main, main, WinMain, libc_start_main, WinMainStartup, start, entry";
+		"main.main, main, wmain, WinMain, wWinMain, DriverEntry, libc_start_main, WinMainStartup, start, entry";
 
 	public enum StartLocationType {
 		LOWEST_ADDRESS("Lowest Address"),
-- 
2.45.1

